tf-psa-crypto/tests/suites/test_suite_psa_crypto.data

```diff
@@ -7925,6 +7925,45 @@ persistent_key_load_key_from_storage:"":PSA_KEY_TYPE_ECC_KEY_PAIR(PSA_ECC_FAMILY
 PSA derive persistent key: HKDF SHA-256, exportable
 persistent_key_load_key_from_storage:"0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b":PSA_KEY_TYPE_RAW_DATA:1024:PSA_KEY_USAGE_EXPORT:0:DERIVE_KEY
 
+PSA interruptible export public key: ECC, SECP256R1, good
```

Comment:
Minor: looking at how the internals look like, it is clear that the negative cases are well tested, but to be on the safe side we could sprinkle in a couple of negative tests as well (eg just to trigger all error branches at least one)